test/arithmetic-test.js

```diff
@@ -599,12 +599,12 @@ describe('BN.js/Arithmetic', function () {
     });
     it('should not allow 0 input', function () {
       assert.throws(function () {
-        BN(1).egcd(0);
+        new BN(1).egcd(0);
```

Comment:
Good catch! Maybe it would be better checking for the actual exception message to fully verify what caused the throw?